doc/examples/edges/plot_convex_hull.py

```diff
@@ -18,38 +15,23 @@
 import matplotlib.pyplot as plt
 
 from skimage.morphology import convex_hull_image
+from skimage import data
+from skimage.util import invert
 
-
-image = np.array(
-    [[0, 0, 0, 0, 0, 0, 0, 0, 0],
-     [0, 0, 0, 0, 1, 0, 0, 0, 0],
-     [0, 0, 0, 1, 0, 1, 0, 0, 0],
-     [0, 0, 1, 0, 0, 0, 1, 0, 0],
-     [0, 1, 0, 0, 0, 0, 0, 1, 0],
-     [0, 0, 0, 0, 0, 0, 0, 0, 0]], dtype=float)
-
-original_image = np.copy(image)
+# The original image is inverted as the object must be white.
```

Comment:
I just think that it is worthwhile to mention the correspondence between intensities and colors here.
From the first read this sentence was very unclear to me: object be white -> it was black? -> what does it mean to be white? -> what does invert actually do?. I think if you could clarify this a bit, it wil be perfect!